sdk/assemblyscript/examples/neo4j/assembly/index.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * This example is part of the Modus project, licensed under the Apache License 2.0.
+ * You may modify and use this example in accordance with the license.
+ * See the LICENSE file that accompanied this code for further details.
+ */
+
+import { neo4j } from "@hypermode/modus-sdk-as";
+
+// This host name should match one defined in the modus.json manifest file.
+const hostName: string = "my-database";
+
+class Person {
+  name: string;
+  age: number;
+  friends: string[] | null;
+
+  constructor(name: string, age: number, friends: string[] | null = null) {
+    this.name = name;
+    this.age = age;
+    this.friends = friends;
+  }
+}
+
+export function CreatePeopleAndRelationships(): string {
+  const person = new neo4j.Variables();
+
+  const people: Person[] = [
+    new Person("Alice", 42, ["Bob", "Peter", "Anna"]),
+    new Person("Bob", 19),
+    new Person("Peter", 50),
+    new Person("Anna", 30),
+  ];
+
+  for (let i = 0; i < people.length; i++) {
+    person.set<string>("name", people[i].name);
+    person.set<number>("age", people[i].age);
+    if (people[i].friends) {
+      person.set<string[]>("friends", people[i].friends!);
+    }
+    const createPersonQuery = `
+      MATCH (p:Person {name: $person.name})
+                UNWIND $person.friends AS friend_name
+                MATCH (friend:Person {name: friend_name})
+                MERGE (p)-[:KNOWS]->(friend)
+    `;
+    const result = neo4j.executeQuery(hostName, createPersonQuery, person);
+    if (!result) {
+      throw new Error("Error creating person.");
+    }
+  }
+
+  return "People and relationships created successfully";
+}
+
+export function GetAliceFriendsUnder40(): neo4j.Node[] {
```

Comment:
I don't think the example should return `Node` objects.  That's the low-level primitive that the database returns, but it's not what the user's own API would be expected to emit.  If the nodes represent people, then something should convert the node back to a `Person` object before returning a `Person[]`